src/inference/src/ie_core.cpp

```diff
@@ -1712,6 +1712,11 @@ Parameter Core::GetConfig(const std::string& deviceName, const std::string& name
         }
     }
 
+    if (name == CONFIG_KEY(FORCE_TBB_TERMINATE)) {
```

Comment:
Where is change for SetConfig() which like GetConfig() here?

Reply:
SetConfig shared the same code with set_property.